pkg/snapshotter/volumesnapshotter.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 Hewlett Packard Enterprise Development LP
+
+package snapshotter
+
+import (
+	"fmt"
+	"github.com/hpe-storage/common-host-libs/storageprovider"
+	"github.com/hpe-storage/common-host-libs/storageprovider/csp"
+	"github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type volumesnapshotter struct {
+	Snapshotter
+	Log             logrus.FieldLogger
+	storageProvider *csp.ContainerStorageProvider
+}
+
+func (v *volumesnapshotter) Init(config map[string]string) error {
+	v.Log.Infof(">>> Init hpe volumesnapshotter with config %+v", config)
+	defer v.Log.Infof("<<<< Init")
```

Comment:
nit: let's keep the number of ">" and "<" consistent :)

I think we use 5 everywhere

Reply:
will update